crates/wash-cli/src/call.rs

```diff
@@ -126,77 +116,82 @@ pub async fn handle_command(
         "calling component function over wRPC"
     );
 
-    let nc = create_nats_client_from_opts(
-        &opts.rpc_host,
-        &opts.rpc_port,
-        opts.rpc_jwt.clone(),
-        opts.rpc_seed.clone(),
-        opts.rpc_credsfile.clone(),
-        opts.rpc_ca_file.clone(),
-    )
-    .await?;
-
-    let mut headers = async_nats::HeaderMap::new();
-    headers.insert("source-id", "wash");
-
-    let lattice = opts
-        .lattice
-        .clone()
-        .unwrap_or_else(|| DEFAULT_LATTICE.to_string());
-
-    // TODO: Configure invocation timeouts
-    let wrpc_client = wasmcloud_core::wrpc::Client::new(
-        nc,
-        &lattice,
-        &component_id,
-        headers,
-        Duration::from_secs(10),
-    );
-
-    let (namespace, package, interface, name) = parse_wit_meta_from_operation(&function).context(
-        "Invalid function supplied. Must be in the form of `namespace:package/interface.function`",
-    )?;
-    let instance = format!("{namespace}:{package}/{interface}");
-    let name = name.context(
-        "Invalid function supplied. Must be in the form of `namespace:package/interface.function`",
-    )?;
-    debug!(
-        ?component_id,
-        ?instance,
-        ?name,
-        ?lattice,
-        "invoking component"
-    );
+    // TODO: Reenable
```

Comment:
Should we remove the call command for now? cc @brooksmtownsend

Reply:
Option 2: Fast follow before release to reenable wash call

Reply:
I believe @vados-cosmonic is already working on updating this?